readthedocs/projects/tasks.py

```diff
@@ -571,6 +578,25 @@ def run_build(self, docker, record):
 
         if self.build_env.failed:
             self.send_notifications(self.version.pk, self.build['id'])
+            # send build failure status to git Status API
+            send_external_build_status(
+                version=self.version, build_pk=self.build['id'], status=BUILD_STATUS_FAILURE
+            )
+        elif self.build_env.successful:
+            # send build successful status to git Status API
+            send_external_build_status(
+                version=self.version, build_pk=self.build['id'], status=BUILD_STATUS_SUCCESS
+            )
+        else:
+            msg = 'Unhandled Build Status'
```

Comment:
I suppose we should communicate GitHub as a FAILURE here. Otherwise, it will be always in yellow "waiting for another status" to display when we already know that it failed.